src/bits/bit_array.rs

```diff
@@ -24,6 +27,7 @@ macro_rules! bit_array_impl {
             impl SharedValue for $name {
                 const BITS: u32 = $bits;
                 const ZERO: Self = Self(<$store>::ZERO);
+                const ONE: Self = Self(bitvec::bitarr!(const Container, Order; 1; $bits));
```

Comment:
I think that we avoided doing this because it requires that we consider the bit array to be something other than an ordered collection of bits.  This explicitly says that the bit array is a representation of a number in a particular order.

I think that's OK, but it's worth documenting that assumption somewhere..